modules/sdk-coin-osmo/test/unit/osmo.ts

```diff
@@ -97,6 +97,16 @@ describe('OSMO', function () {
       should.equal(utils.isValidValidatorAddress(undefined as unknown as string), false);
       should.equal(utils.isValidValidatorAddress(''), false);
     });
+
+    it('should all type of address on Osmo coin address validation', () => {
+      const tomso = bitgo.coin('tosmo');
+      should.equal(tomso.isValidAddress(address.address1), true);
+      should.equal(tomso.isValidAddress(address.validatorAddress1), true);
+      should.equal(tomso.isValidAddress(address.contractAddress1), true);
+      should.equal(tomso.isValidAddress('dfjk35y'), false);
+      // @ts-expect-error - testing invalid address
+      should.equal(tomso.isValidAddress(undefined), false);
+    });
```

Comment:
it('should validate account addresses correctly', () => {
}

check this above. Add contractAddress also there itself. No need for a new case.

Reply:
there we are checking `utils.isValidAddress` that only check for account address, it will fail for validator address and contract address. I think we should rename it to isValidAccountAddress.
In my new test case I am checking `osmo.isValidAddress` that works for all three kinds of addresses

Reply:
Got it, so
Osmo.isValidAddress() contains all three types of check that is there in utils file.

So its more or less the same thing.

But instead of repeated checks, can we just have some cases for isValidContractAddress() particularly.

If you want you can keep this as well. Just change the description that is easy to understand.